inc/minishell_config.h

```diff
@@ -19,6 +19,8 @@
 # define MAX_SHLVL 10
 # define BUF_SIZE 2048
 
+# define PS1 "PS1=\\u@\\h:\\w$ "
```

Comment:
We are in another context: prompt creation. Here we don't have shell variables like $USER. We have predefined escape sequences like \u and \w. 

This means u is just the letter u. But escaped with backslash: \u it will be replaced with the variable $USER 
(btw real bash uses getuid() or something and doesn't rely on shell variables - so it's even more removed from shell variables)

$ has no special meaning in this context, not as a letter nor as sequence \$

Reply:
Ok, but why don't we just right \u@\h:\w$ then?

What is the double back slash for?

Reply:
That's what I meant in my first comment: 

so in the prompt env->

u is just u
\u is $USER
\\u is ...?

Reply:
It seems that on GitHub the backslash also has a special meaning. To write two backslashes you need to write four: \\\\

I updated the above comment. I hope it's more clear now.

Reply:
A bit, but let's talk about it when we meet again.